docs/source/standard/auction.rst

```diff
@@ -56,65 +138,76 @@ Schema
    The total estimated value of the procurement.
 
 :guarantee:
-    :ref:`Guarantee`
+   :ref:`Guarantee`, required
 
-    Bid guarantee
+   The assumption of responsibility for payment of performance of some obligation if the liable party fails to perform to expectations.
 
 :items:
-   list of :ref:`item` objects, required
+   Array of :ref:`item` objects, required
+
+   Can be edited during the rectificationPeriod (Can editing in 2 ways: each object from the array separately and the entire array together).
 
    List that contains single item being sold. 
 
    |ocdsDescription|
    The goods and services to be purchased, broken into line items wherever possible. Items should not be duplicated, but a quantity of 2 specified instead.
 
 :features:
-   list of :ref:`Feature` objects
+   Array of :ref:`Feature` objects
 
    Features of auction.
 
 :documents:
-   List of :ref:`document` objects
+   Array of :ref:`document` objects
  
    |ocdsDescription|
    All documents and attachments related to the auction.
 
-:questions:
-   List of :ref:`question` objects
+:dateModified:
+   :ref:`date`, auto-generated, read-only
 
-   Questions to ``procuringEntity`` and answers to them.
+   |ocdsDescription|
+   Date when the auction was last modified
 
-:complaints:
-   List of :ref:`complaint` objects
+:questions:
+   Array of :ref:`question` objects
 
-   Complaints to auction conditions and their resolutions.
+   Questions to ``procuringEntity`` and answers to them.
 
 :bids:
-   List of :ref:`bid` objects
+   Array of :ref:`bid` objects
 
    A list of all bids placed in the auction with information about participants, their proposals and other qualification documentation.
 
    |ocdsDescription|
    A list of all the companies who entered submissions for the auction.
 
 :awards:
-    List of :ref:`award` objects
+    Array of :ref:`award` objects
 
     All qualifications (disqualifications and awards).
 
 :contracts:
-    List of :ref:`Contract` objects
+    Array of :ref:`Contract` objects
+
+    |ocdsDescription|
+    Information on contracts signed as part of a process.
+
+:rectificationPeriod:
+   :ref:`period`, auto-generated, read-only
+
+   The time span during which the owner can edit some data within the procedure. Editing is possible within 48 hours from the inception of the procedure.
```

Comment:
Editing is possible within 48 hours from the inception of the procedure. - to be removed